src/commands/new/glee.ts

```diff
@@ -1,55 +1,39 @@
-import {Flags} from '@oclif/core';
-import { promises as fPromises } from 'fs';
-import Command from '../../base';
-import { resolve, join } from 'path';
-import fs from 'fs-extra';
+import { Flags } from "@oclif/core";
+import Command from "../../base";
 
 export default class NewGlee extends Command {
-  static description = 'Creates a new Glee project';
-
-  protected commandName = 'glee';
+  static description = "Creates a new Glee project";
+  protected commandName = "glee";
 
   static flags = {
-    help: Flags.help({ char: 'h' }),
-    name: Flags.string({ char: 'n', description: 'name of the project', default: 'project' }),
-    template: Flags.string({ char: 't', description: 'name of the template', default: 'default' }),
+    help: Flags.help({ char: "h" }),
+    name: Flags.string({
+      char: "n",
+      description: "name of the project",
+      default: "project",
+    }),
   };
 
+  static args = [
+    {
+      name: "file",
+      description: "spec path, URL or context-name",
+      required: true,
+    },
+  ];
   async run() {
-    const { flags } = await this.parse(NewGlee); // NOSONAR
```

Comment:
we don't want to remove the template support. Users should still be able to create a glee project from one of the templates.

Reply:
ok